frontend/src/metabase/ui/components/inputs/Select/Select.styled.tsx

```diff
@@ -21,9 +21,9 @@ export const getSelectOverrides = (): MantineThemeOverride["components"] => ({
         color: "text-dark",
       },
     }),
-    styles: (theme, _, { size = "md" }) => ({
-      ...getSelectInputOverrides(theme, size),
-      ...getSelectItemsOverrides(theme, size),
+    styles: theme => ({
+      ...getSelectInputOverrides(theme),
+      ...getSelectItemsOverrides(theme),
```

Comment:
I think this might be leading to some errors. I checked /admin/performance and that page doesn't load. Probably something tiny. Maybe the size parameter is needed after all?

Reply:
Yes I was a bit overzealous, fixed the error.